libs/entitlement/src/test/java/org/elasticsearch/entitlement/runtime/policy/PolicyManagerTests.java

```diff
@@ -78,7 +82,7 @@ public void testGetEntitlementsThrowsOnMissingPolicyForPlugin() {
         var callerClass = this.getClass();
         var requestingModule = callerClass.getModule();
 
-        assertEquals("No policy for this plugin", ModuleEntitlements.NONE, policyManager.getEntitlements(callerClass));
+        assertEquals("No policy for this plugin", ModuleEntitlements.NONE, policyManager.getEntitlements(callerClass, Entitlement.class));
```

Comment:
Can we get rid of this class parameter? Looks like it's only used to add some slightly misleading text to a logger warning [here](https://github.com/elastic/elasticsearch/blob/6af3b01044d0e989ac17707a5f5df78d74555988/libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/PolicyManager.java#L300-L301).

Reply:
I added that because it's really valuable to "debug" which policies are missing which entitlements

Reply:
Oh I see what you're after. Good point, let me ponder that. 🤔